test/integration/features/config.feature

```diff
@@ -13,15 +13,15 @@ Checks whether CRC `config set` command works as expected in conjunction with `c
         @darwin
         Examples: Config settings on Mac 
             | property  | value1                | value2                                     |
-            | bundle    | crc_vbox_4.1.0.tar.xz | /path/to/fake/bundle/crc_vbox_4.1.0.tar.xz |
+            | bundle    | crc_vbox_4.1.3.tar.xz | /path/to/fake/bundle/crc_vbox_4.1.3.tar.xz |
```

Comment:
Fwiw, it would be great if we could substitude this 4.1.3 from the Makefile somehow, rather than having to change it in multiple places.

Reply:
@cfergeau, yes that we will do for sure, will create an follow up issue.